### PR TITLE
Add more metadata to RSS-ingested package (versioncreated, guid, etc.)

### PR DESCRIPTION
**Pull request by plamut**
It turned out that RSS packages lacked metadata and were filtered out because they were recognized as expired, and they also lacked the `guid` attribute. This PR fixes that.

### CODE REVIEW

File: server/superdesk/io/rss.py [hunk 1]
```diff
@@ -307,6 +310,9 @@ def _create_package(self, text_item, image_items):
         """
         package = {
             'type': 'composite',
+            'guid': 'package:' + text_item['guid'],
```
Comment (plamut) [inline server/superdesk/io/rss.py]: Is this an OK guid for a package?

Comment (amagdas) [inline server/superdesk/io/rss.py]: IMO we should use this: https://github.com/superdesk/superdesk/blob/master/server/apps/archive/common.py#L88

**Comment by mugurrus** [comment]
:+1: but use what Adrian suggested for guid generation

File: server/superdesk/io/rss.py (hunk 1 above)
Comment (plamut) [inline server/superdesk/io/rss.py]: This is exactly what I was looking for, thanks. I'll update.

Comment (plamut) [inline server/superdesk/io/rss.py]: BTW, what to use for the "hint" to the linked function, simply the GUID_TAG from the settings?

**Comment by plamut** [comment]
Updated, `generate_guid()` is now used for items' and packages' `guid` field values.

**Comment by amagdas** [comment]
:+1:

**Pull request merged**